docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.9.12-slim-buster
+ARG VERSION
+
+WORKDIR /app
+
+COPY run.sh .
+
+# https://askubuntu.com/a/649463
```

Comment:
Could you add a short description - why the link is useful and how it relates to commands?